webpack.config.js

```diff
@@ -29,4 +29,4 @@ module.exports = {
   optimization: {
```

Comment:
- [ ] Data is not being saved on local storage, kindly check into that on why is happening, there are few issues with it

Reply:
- [ ] Because of no local storage I couldn't test the following but make sure to implement them:
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..